src/Subscriber.ts

```diff
@@ -247,18 +247,18 @@ class SafeSubscriber<T> extends Subscriber<T> {
     }
   }
 
-  private __tryOrUnsub(fn: Function, value?: any): void {
+  private __tryOrUnsub(fn: Function, ...value: Array<any>): void {
     try {
-      fn.call(this._context, value);
+      fn.call(this._context, ...value);
```

Comment:
@kwonoj couldn't this be `fn.apply(this._context, value)`? And since this is a hot code path, can we avoid allocating the rest args Array with a third boolean argument as a flag?